browser/ui/webui/brave_new_tab_page_refresh/brave_new_tab_page_ui.cc

```diff
@@ -48,7 +50,7 @@ using brave_new_tab_page_refresh::VPNFacade;
 }  // namespace
 
 BraveNewTabPageUI::BraveNewTabPageUI(content::WebUI* web_ui)
-    : ui::MojoWebUIController(web_ui) {
+    : ui::MojoWebUIController(web_ui, /* enable_chrome_send = */ true) {
```

Comment:
interesting, so we need to add this param in order to use the `PluralStringHandler`?

cc @thypon, are there any security concerns around using this (noting that we already do this on the existing NTP)

Reply:
That's right. `PluralStringHandler` is an old-school message handler. The other alternative would be to add a `GetPluralString` method to the `NewTabPageHandler` interface (this approach was taken in `RewardsPageHandler`). Either way works for me.

Reply:
Nah makes sense - plumbing it through mojom just adds another layer of indirection.